src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx

```diff
@@ -19,9 +19,21 @@ export const GetStarted: React.FunctionComponent = withErrorBoundary(() => (
       </Paragraph>
     </TopSection>
     <BottomSection>
-      <Button size="large" category="secondary">
-        <Link to="/Signup">Sign-Up</Link>
-      </Button>
+      {localStorage.getItem('authenticated') ? (
+        <a
+          href="https://app.slack.com/client/TRN1H1V43/CUCLB9VJL"
+          target="_blank" 
+          rel="noopener noreferrer"
+        >
+          <Button category="secondary" size="large">
```

Comment:
Please change the category of a button to `primary` as seen here http://www.ignitus.org/